xtext-website/download.html

```diff
@@ -19,7 +19,7 @@ <h2>Download Options</h2>
 			</div>
 		</div>
 		<div class="row">
-			<div class="span6">
+			<div class="span12">
 				<h2>Eclipse Users</h2>
```

Comment:
This headline becomes unnecessary since there are only Eclipse users. Maybe remove or reword it?